paddle/fluid/platform/nccl_helper.h

```diff
@@ -124,8 +124,8 @@ struct NCCLContextMap {
           } else {
             rank = trainer_id;
           }
-          VLOG(3) << "init nccl rank: " << rank << " nranks: " << nranks
-                  << " gpu id: " << gpu_id;
+          VLOG(1) << "init nccl rank:" << rank << ", nranks:" << nranks
+                  << ", gpu_id:" << gpu_id << ", dev_id:" << order_[i];
```

Comment:
Why change this ?

Reply:
It's important info for user to check if the nccl run ok or not!